IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricFlowStorageDeviceTypeCapacitor/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of capacitor components used in railway. The property set can be used by the predefined type CAPACITOR of IfcElectricFlowStorageDevice
```

Comment:
Only railway?

Reply:
Not only. 0871f7b28802c8b6bf8a2b767b0356cb11e25edd